src/arch/ofi/conv-common.h

```diff
@@ -91,11 +91,11 @@
 #define CMK_USE_PMI                                     1
 #define CMK_USE_PMI2                                    0
 #define CMK_USE_PMIX                                    0
-
+#define CMK_USE_CRAYPMI                                 0
 /*
  * Use Simple client-side implementation of PMI.
  * Valid only for CMK_USE_PMI.
  * Optional in an SLURM environment.
  * See src/arch/util/proc_management/simple_pmi/
  */
-#define CMK_USE_SIMPLEPMI                               1
+#define CMK_USE_SIMPLEPMI                               0
```

Comment:
Does this get set somewhere else now in the default OFI case?

Reply:
I'm not sure if that was a good to have.  It seems to me that it may be better to have the builder choose their pmi on the build line.  Simplepmi will often compile just fine, but may not run launch the way the user expects.  I've seen it result in a non parallel launch (as in a bunch of independent processes), which is arguably worse.